.pre-commit-config.yaml

```diff
@@ -1,6 +1,6 @@
 repos:
 - repo: https://github.com/psf/black
-  rev: 21.7b0
+  rev: 22.3.0
```

Comment:
Do we also need to update the requirements.txt file?

Reply:
Yeah, that's a good idea.